"2019-08-24 04:42:40,484 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:42:40,484 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:42:40,560 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:42:40,560 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:42:51,825 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:42:51,825 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:42:51,825 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:42:51,825 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:42:55,144 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/a1c7ddc4-c629-11e9-9de3-0242ac110002/0', but this can be ignored sometimes"
"2019-08-24 04:43:03,863 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:03,863 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:03,867 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:03,868 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:15,504 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:15,504 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:15,513 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:15,513 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:26,722 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:26,722 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:26,734 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:26,735 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:37,870 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:37,870 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:38,151 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:38,151 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:40,884 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 04:43:49,310 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:49,311 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:49,652 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:43:49,652 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:01,117 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:01,117 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:01,125 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:01,125 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:17,093 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:17,093 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:17,098 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:17,098 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:32,842 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:32,842 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:33,163 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:33,163 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:54,972 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:54,972 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:54,995 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:44:54,995 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:05,778 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/ec465664-c629-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 04:45:14,443 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:14,443 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:14,605 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:14,606 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:17,855 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 04:45:26,534 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:26,534 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:26,548 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:26,548 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:38,512 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:38,512 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:38,520 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:38,520 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:50,340 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:50,340 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:50,540 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:45:50,540 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:02,212 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:02,212 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:02,883 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:02,883 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:14,081 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:14,081 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:14,094 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:14,094 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:20,645 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:20,645 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:20,656 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:20,656 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 04:46:23,721 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 04:48:07,145 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 04:48:30,685 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/69e59698-c62a-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 04:49:41,036 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 04:49:47,693 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 04:50:44,023 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 17] File exists: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/b94cbaea-c62a-11e9-9de3-0242ac110002/0', but this can be ignored sometimes"
"2019-08-24 04:51:18,237 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 04:51:41,681 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 04:53:43,437 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 04:53:55,061 - data_overview.py[line:90] - WARNING: destroy Dtable error,:/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/2b536ce2-c62b-11e9-9de3-0242ac110002/0: No such file or directory, but this can be ignored sometimes"
"2019-08-24 04:54:06,534 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/3223eb96-c62b-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 04:54:17,318 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 04:54:52,429 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/4ce9d90e-c62b-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 04:55:49,701 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 04:56:01,848 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/76961fa6-c62b-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 04:56:25,155 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 04:56:31,288 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 04:57:28,471 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 04:58:14,116 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 04:59:00,049 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/e0ec8458-c62b-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 04:59:52,745 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:02:11,086 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:03:15,213 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:03:27,119 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:03:49,615 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/8da60322-c62c-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:04:35,726 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/a8f2741c-c62c-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:04:47,124 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/b00f1a2a-c62c-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:05:10,171 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 05:07:00,046 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/fef29540-c62c-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:07:23,162 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:08:08,665 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/2799133e-c62d-11e9-9de3-0242ac110002/0', but this can be ignored sometimes"
"2019-08-24 05:08:19,953 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:08:31,462 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/35ebf140-c62d-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:08:42,895 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 05:09:17,719 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/50d2a83c-c62d-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:09:52,910 - data_overview.py[line:90] - WARNING: destroy Dtable error,:/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/65e143dc-c62d-11e9-9de3-0242ac110002/0: No such file or directory, but this can be ignored sometimes"
"2019-08-24 05:10:21,861 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:11:30,478 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:11:53,799 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/ae0db078-c62d-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:12:16,767 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/bba9689e-c62d-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:13:03,341 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/d77d8cb2-c62d-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:13:21,927 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:14:08,962 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:14:54,587 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 05:15:29,624 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/2ed1d04a-c62e-11e9-9de3-0242ac110002/0', but this can be ignored sometimes"
"2019-08-24 05:16:05,386 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:17:09,212 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:18:18,764 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:20:31,972 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:20:43,194 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/e9fcb2b8-c62e-11e9-9de3-0242ac110002/0', but this can be ignored sometimes"
"2019-08-24 05:21:28,607 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:23:13,891 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/435e4af6-c62f-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:23:42,891 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/551ab068-c62f-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:24:28,288 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:25:25,152 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 05:25:36,935 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:25:48,034 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:26:52,761 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:27:15,762 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:27:39,131 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:27:50,550 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/e87d7ffc-c62f-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:28:12,994 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/f5cd6a64-c62f-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:28:36,047 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:30:09,458 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/3b3496a4-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:30:15,821 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/3f28f822-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:30:27,722 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 05:30:39,297 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/4d23324e-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:30:50,676 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/53e1599e-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:31:13,440 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/61b35784-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:31:59,517 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/7cad34ce-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:32:22,482 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/8a6f82a6-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:32:45,551 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:33:49,512 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/be9f90de-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:35:10,537 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/ee9dfe6a-c630-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:36:19,827 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/18032618-c631-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:36:31,339 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/1ee57620-c631-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:37:57,989 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/52d242ec-c631-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:38:32,879 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/674f1c18-c631-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:38:56,022 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:39:19,051 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/82b05850-c631-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:40:34,930 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 05:40:57,985 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:41:32,492 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/d2744b4e-c631-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:41:55,702 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:43:05,968 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-24 05:43:46,969 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/22ce3212-c632-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:43:59,189 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:44:21,865 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/3753e56a-c632-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:44:45,159 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:45:07,320 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/5297ddb8-c632-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:45:18,752 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:45:41,696 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 05:46:27,217 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/81cd39c0-c632-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:46:38,583 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:47:07,549 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/9a3c4bd6-c632-11e9-9de3-0242ac110002/0', but this can be ignored sometimes"
"2019-08-24 05:47:18,986 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/a13ac7be-c632-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:47:41,843 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/ae7b62bc-c632-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:48:15,657 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:48:38,376 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-24 05:49:12,642 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/e4b1d2c6-c632-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:49:35,795 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824044216/f24d8eca-c632-11e9-9de3-0242ac110002', but this can be ignored sometimes"
"2019-08-24 05:50:21,440 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
